addons/website/models/ir_ui_view.py

```diff
@@ -93,6 +93,54 @@ def write(self, vals):
 
         return True
 
+    @api.multi
+    def _get_specific_views(self):
+        """ Given a generic view (no website_id), return a record set containing
```

Comment:
As we discussed, it also works if it's called on a specific view

Reply:
Yeah that docstring was stinky, adapted it to reality